### ISSUE TITLE
Working on Cloud SQL and Cloud Storage integration

### ISSUE THREAD

**Issue by davisshaver** [daxio/k8s-lemp#14]
Hi there! This is _exactly_ what I had been looking to build myself. Going to clone this morning and try with your setup instead. My key requirement is Cloud SQL and Cloud Storage integration for horizontal scalability of the clusters. Let me know what you think, maybe a PR will make sense.

**Comment by chepurko** [comment]
@davisshaver for sure! Thanks for your interest.

If I may ask, have you tried the non-open-source [Google Container Engine (GKE)](https://cloud.google.com/container-engine/) for this yet? The reason I'm asking is the pure open-source Kubernetes commonly runs the open source MySQL in a container with plain old block storage.

I believe GKE has drop-in functionality for Google's Cloud SQL and Cloud Storage buckets. But I'm not aware if storage buckets are built for on-demand storage as it's intended for object storage (like Google Drive, for example) rather than block storage.

**Comment by davisshaver** [comment]
@chepurko Yes I intend to host on GKE, although I'm intent on having everything work on Minikube, too. Have you tested this setup with Minikube? 

Re Cloud Storage, I am using it to host `uploads` folder in the same way as one might use s3.

**Comment by davisshaver** [comment]
@chepurko I was wondering if I could contact you via email, I am trying to find freelance help with k8s and WordPress and was wondering if you were available or had any recommendations...

**Comment by chepurko** [comment]
Sure, want to join me on Keybase? I already pinged your GitHub account there. You’ll see it as soon as you join and verify your GitHub account.

**Comment by vinceano** [comment]
Totally lost; need help ...S.O.S..!
